src/js/connection/drivers/http.js

```diff
@@ -55,17 +55,15 @@ export default {
     },
 
     getCSRFToken() {
-        const tokenTag = document.head.querySelector('meta[name="csrf-token"]')
-        let token
-
-        if (!tokenTag) {
-            if (!window.livewire_token) {
-                throw new Error('Whoops, looks like you haven\'t added a "csrf-token" meta tag')
-            }
+        const { content: token } = {
+            ...{
+                content: window.livewire_token || null
+            },
+            ...document.head.querySelector('meta[name="csrf-token"]'),
+        }
```

Comment:
Going to be honest, I have a really hard time following that refactor. Even though the original is a bit ugly, I think I still prefer it for readability.